.circleci/config.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       SEMGREP_REPO_URL: << pipeline.project.git_url >>
       SEMGREP_BRANCH: << pipeline.git.branch >>
     docker:
-      - image: returntocorp/semgrep
+      - image: cimg/deploy:2023.09-node
```

Comment:
I guess you need to change this one image https://github.com/verygoodsecurity/vgs-collect-js/blob/d880ad8ee76a311eafe0ed345f0025d07a241d6f/.circleci/config.yml#L4C3-L4C28

for now, you changed docker image for Semgrep security scanner :)

Reply:
Thanks for the heads up